synapse/handlers/identity.py

```diff
@@ -803,6 +811,9 @@ async def ask_id_server_for_third_party_invite(
             "sender_display_name": inviter_display_name,
             "sender_avatar_url": inviter_avatar_url,
         }
+        # If a custom web client location is available, include it in the request.
+        if self._web_client_location:
+            invite_config["web_client_location"] = self._web_client_location
```

Comment:
I think we probably want to namespace non-standard parameters with `org.matrix.`?

Reply:
Oh, that's a good call. 👍